plugins/plugin-site/src/templates/plugin.jsx

```diff
@@ -21,11 +21,14 @@ import PluginReleases from '../components/PluginReleases';
 import PluginIssueTrackers from '../components/PluginIssueTrackers';
 
 function shouldShowWikiUrl({url}) {
-    return url?.startsWith('https://wiki.jenkins-ci.org/') || url?.startsWith('https://wiki.jenkins.io/') || url?.includes('github.com/jenkins-infra/plugins-wiki-docs');
+    return url?.startsWith('https://wiki.jenkins-ci.org/') ||
+        url?.startsWith('https://wiki.jenkins.io/') ||
+        url?.includes('github.com/jenkins-infra/plugins-wiki-docs') ||
+        url?.includes('raw.githubusercontent.com/jenkins-infra/plugins-wiki-doc');
 }
 
 function shouldShowGitHubUrl({url}) {
-    return url && url.startsWith('https://github.com/');
+    return url?.startsWith('https://github.com/') && !shouldShowGitHubUrl(url);
```

Comment:
this would be an infinite recursion but the missing brackets prevent it
```suggestion
    return url?.startsWith('https://github.com/') && !shouldShowWikiUrl({url});
```

Reply:
oops, thanks, i sorta just rushed it.